lib/flutter_billing.dart

```diff
@@ -37,6 +39,9 @@ class BillingProduct {
   /// Price in 100s. e.g. $2.49 equals 249.
   final int amount;
 
+  // Type of product. e.g. SUBS or INAPP
+  final String type;
```

Comment:
Prefer to use [enum](https://www.dartlang.org/guides/language/language-tour#enumerated-types) instead